pkg/cluster/flags/debug.go

```diff
@@ -24,7 +24,6 @@ var (
 )
 
 func init() {
-	if s := os.Getenv("TIUP_CLUSTER_DEBUG"); s == "enable" {
-		DebugMode = true
-	}
+	ShowBacktrace = len(os.Getenv("TIUP_BACKTRACE")) > 0
```

Comment:
For the env which can be modify by user, better to define it in [localdata](https://github.com/pingcap/tiup/blob/master/pkg/localdata/constant.go#L26) and list it in [envList](https://github.com/pingcap/tiup/blob/master/cmd/env.go#L24)

Reply:
I got it

Reply:
BTW, `ShowBacktrace` was not used, I'll remove it.

Reply:
one more thing, `tiup env` currently only displays the env with  `key=val`, should we extend it with `key=val description`, the description will tell users how to set the env, or the usage of this env?

Reply:
Maybe the  current style (same with go env) is more friendly for script?